### ISSUE TITLE
question about rnnt result

### ISSUE THREAD

**Issue by jinggaizi** [hirofumi0810/neural_sp#270]
hi,
i find configuration neural_sp/examples/librispeech/s5/conf/asr/transducer/lc_transformer_rnnt_64_64_32_bpe1000.yaml. What is the result of this configuration
thanks

**Comment by hirofumi0810** [comment]
@jinggaizi No. I will not report it. Instead I'll report results with streaming Conformer encoder soon.

**Comment by jinggaizi** [comment]
Is decoder transducer?
